server.js

```diff
@@ -13,6 +15,7 @@ const localConfigExists = fs.existsSync(
 const serverConfig = {
   contentBase: config.output.path,
   publicPath: config.output.publicPath,
+  disableHostCheck: process.env.PERFHTML_PUBLICHOST === 'true',
```

Comment:
remove this and add `public: host`

Reply:
Removed. But is `public: host` still needed here? Or maybe `host` should be added to `allowedHosts`?